module/support/logging/DataLogging/data/config/DataLogging.yaml

```diff
@@ -6,7 +6,18 @@ output:
 
 # You can put any messages that is emitted in the system here and it will be logged
 # The below ones are just the most common and here as an example
+# You can put any messages that is emitted in the system here and it will be logged
 messages:
-  message.output.CompressedImage: false
+  message.support.nusight.Overview: false
```

Comment:
is there any difference between this and the webots config? if not then delete the webots one